packages/dialtone-vue2/recipes/conversation_view/emoji_row/emoji_row.vue

```diff
@@ -159,7 +159,10 @@ export default {
   }
 
   &__reaction-number {
-    font: var(--dt-typography-body-small);
+    // font: var(--dt-typography-body-sm); // replace with this once Style Dictionary transform resolved
```

Comment:
Is there a ticket for fixing the style dictionary?

Reply:
Yeah: https://dialpad.atlassian.net/browse/DLT-1612

I was going to call it out to Brad and Julio since they're done most of the Style Dictionary stuff. I wasn't sure if it were straight-forward or needs a little care. I tried to figure it out, but probably shouldn't dare. Whatchathink @braddialpad?

It doesn't block this task, though would be lovely to resolve soon.

Reply:
I'll add that mapping to https://dialpad.atlassian.net/jira/software/c/projects/DLT/issues/DLT-1612. Would you prefer to piggy-pack that change onto this PR, or its own task after this one merges?

Reply:
Yeah I'll just tack it on here, since we probably shouldn't merge this without it

Reply:
Sweet. I've added the mapping in the description field of https://dialpad.atlassian.net/browse/DLT-1612.

Additionally, I'm assuming this is the same culprit for why it doesn't appear [on the Tokens table](https://dialtone.dialpad.com/deploy-previews/pr-198/tokens/#typography). Just like for the CSS Variable, the typography property is available (e.g. `var(--dt-typography-body-md-font-family)`) but not the overall composition one, (e.g. `var(--dt-typography-body-md)`.

The best way to test your PostCSS update is the remove all the lines that have 
"`// remove once Style Dictionary transform resolved`".